### PR TITLE
Bump dependencies + fixes

### PR DESCRIPTION
**Pull request by santiagofn**
- Removed `file-loader` & `url-loader` and replaced them with webpack5 `asset` (https://webpack.js.org/guides/asset-modules/)
- Removed `react-bootstrap-table-next` (no longer maintained), now using `react-bootstrap/Table`
- Bump `style-loader`, `postcss-loader`, `css-loader`
- Fixed bootstrap generic CSS creation

### CODE REVIEW

File: components/shared/Table.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
-import BootstrapTable from 'react-bootstrap-table-next';
+import Table from 'react-bootstrap/Table';
```
Comment (akshaaatt) [inline components/shared/Table.js:1]: @santiagofn this breaks the storybook example at `stories/index.stories.js`. Can you please fix it?

**Review by akshaaatt — CHANGES_REQUESTED** [review]
Thanks a lot for the updates! 💯

File: package.json [hunk 2]
```diff
@@ -53,29 +53,29 @@
...
     "react-dom": "^17.0.2",
     "sass": "^1.55.0",
     "sass-loader": "^13.0.2",
     "storybook-addon-designs": "^6.2.1",
-    "style-loader": "^0.21.0",
-    "url-loader": "^1.0.1",
+    "style-loader": "^3.3.2",
     "webpack": "^5.74.0"
+  },
+  "dependencies": {
```
[the remark this answers is not in the document]
Comment (santiagofn) [reply package.json:78]: My bad, it slipped in the PR. Will remove it.

[the remark this answers is not in the document]
Comment (santiagofn) [reply package.json:78]: My bad, it slipped in the PR. Will remove it.

File: components/shared/Table.js (hunk 1 above)
Comment (santiagofn) [reply components/shared/Table.js:1]: My bad 2, didn't notice this! Fixed now.

**Review by akshaaatt — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**